components/sysutils/neofetch/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,26 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Benjamin S. Osenbach
+#
```

Comment:
You have mixed the places for the Copyright. The sample-manifst should stay as it is generated (it's both, a template for the real manifest file and a baseline to check if something has changed in a new version).